confdgnmi/src/confd_gnmi_adapter.py

```diff
@@ -289,7 +289,7 @@ def get_subscription_handler(self,
     def get_inst(cls):
         """
         Get adapter instance
-        We use this, since we want to have adapter as singleton.
+        We use this, since we want to have some adapter variants as singleton.
```

Comment:
Good, but the `get_inst` name is most frequently used for singleton classes to return the instance; if that's not the case, I guess something like `get_adapter` might be better.

Reply:
Why not :-), will be part of next release.